.github/workflows/rust.yml

```diff
@@ -0,0 +1,50 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{matrix.os}}
+
+    strategy: 
+      matrix:
+        rust: [stable]
```

Comment:
We need to make sure that we're building with an MSRV (Minimal Supported Rust Version, which is 1.34.0 for this crate) compiler instead of the latest stable. See our Travis CI manifest for how it's done currently.

Reply:
Yeah, I saw that `export MSRV=1.34.0`, This not work on windows, so we need to use `set MSRV=1.34.0` on windows?

Reply:
Yeah, since it's just a variable, whichever way to declare it works should be used on windows.